client/src/components/Card/Card.css

```diff
@@ -105,24 +116,29 @@
   -moz-box-shadow: none;
   box-shadow: none;
 }
+
 .Card__discover-link {
   color: black;
   text-align: center;
   transition: transform 0.3s;
 }
+
 .Card__discover-link a {
   font-size: 24px;
   width: 10rem;
   margin-left: 2rem;
   margin-right: 2rem;
   text-decoration: none;
 }
+
 .Card__discover-link:hover {
   transform: scale(1.1);
 }
+
 .Card__discover-link a:hover {
   color: var(--yellow);
 }
```

Comment:
This seems like contextual information when we want these to be abstracted. Is there a better way of naming this?